src/main/java/net/mcreator/plugin/PluginLoader.java

```diff
@@ -68,14 +71,20 @@ public PluginLoader() {
 		super(new URL[] {}, null);
 
 		this.plugins = new ArrayList<>();
+		this.javaPlugins = new ArrayList<>();
 		this.pluginUpdates = new ArrayList<>();
 
+		DynamicURLClassLoader javaPluginCL = new DynamicURLClassLoader(new URL[] {}, Thread.currentThread().getContextClassLoader());
+
 		UserFolderManager.getFileFromUserFolder("plugins").mkdirs();
 
 		List<Plugin> pluginsLoadList = new ArrayList<>();
 		pluginsLoadList.addAll(listPluginsFromFolder(new File("./plugins/"), true));
 		pluginsLoadList.addAll(listPluginsFromFolder(UserFolderManager.getFileFromUserFolder("plugins"), false));
 
+		if (System.getenv("MCREATOR_PLUGINS_FOLDER") != null)

```

Comment:
If the path is missing, maybe you should write something inside the logs, so the user knows his plugin needs something.

Reply:
This path is only used by the development gradle script. It is also completely normal for it to be absent, in fact it is default state

Reply:
👍

Reply:
👍